index.bs

```diff
@@ -9980,13 +9981,18 @@ dictionary AudioParamDescriptor {
 	float defaultValue = 0;
 	float minValue = -3.4028235e38;
 	float maxValue = 3.4028235e38;
+	AutomationRate automationRate = "a-rate";
```

Comment:
So by default, it's `a-rate`?

Reply:
Chrome is a-rate now, so for compatibility, use "a-rate" as the default.